torchci/lib/JobClassifierUtil.ts

```diff
@@ -215,8 +215,10 @@ export function getGroupConclusionChar(conclusion?: GroupedJobStatus): string {
       return "O";
     case GroupedJobStatus.Failure:
       return "X";
+    case GroupedJobStatus.Queued:
+      return "Q";
     case GroupedJobStatus.Pending:
-      return "?";
+      return "P";
```

Comment:
I would prefer to keep ? here (until you change it properly to use react icons), people are familiar with seeing that for pending jobs

Reply:
Hi Huy, I'm thinking replace those with react-icons in next PR, but def, i can change it back to this pr

Reply:
Yeah, I think this change goes beyond scope of this PR which is to add the queued status.  So, it's better to save it on the next PR as you plan to replace those anyway.  We can ask more people to look at the next PR to gather their opinions

Reply:
sounds good! Will add both Q and this change to what you suggest